src/graphql-server/resolvers.js

```diff
@@ -89,6 +89,8 @@ export const resolvers = {
 
     updateHostPricing: (_, { units, pricePerUnit }) => HhaDnaInterface.hostPricing.update(units, pricePerUnit),
 
+    holofuelUpdateUser: (_, { id, nickname, avatarUrl }) => HoloFuelDnaInterface.user.update(id, nickname, avatarUrl),
```

Comment:
This is just so we can return the full profile back, once completing a successful zome call, as the dna returns an address.

Reply:
Ok. We decided that we will extract this id and just make a separate call to the dna to get the agent hash, in order to clean the ui and prepare for the dna update that returns back the full profile instead of just the hash address.